src/resources/SpecialResolution/CooperativeResource.ts

```diff
@@ -20,7 +20,24 @@ export const CooperativeResource: ResourceIF = {
     typeChangeInfo: 'You cannot change the business type of a Cooperative Association. You must form a new' +
     'business and dissolve this business once the new business is registered.',
     addressChangeInfo: 'To change addresses, please use the Change feature in the' +
-    ' Office Addresses list on your business dashboard.'
+    ' Office Addresses list on your business dashboard.',
+    specialSpecialResolution: {
+      helpSection: {
+        header: 'Help with Special Resolution',
+        helpText: [
+          'If your require assistance with adding a business partner please contact us.'
```

Comment:
Should be "If you require".

Reply:
this is the same for SP/GP also. I copied it from Sp/GP resource file. Can I change all places?

Reply:
Oops! Yes, please fix all 3 :)